src/extension.ts

```diff
@@ -11,6 +12,9 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	context.workspaceState.update('enabled', false);
 	vscode.debug.registerDebugConfigurationProvider('*', new ConfigurationProvider(), 2);
+	
+	// start mirrord binary updates, so that we avoid downloading mid session
+	await getMirrordBinary(true);
```

Comment:
@Razz4780 Was there a reason we didn't do this originally?

Reply:
@infiniteregrets So why is it necessary now?

Reply:
because downloading mid-session timed out the debug session for me

Reply:
I rested a few times with a manual timeout, its not happening anymore now? might be something else that I encountered

Reply:
I think we can still keep this change, as it still helps make the extension faster in most cases, @Razz4780 wdyt